pages/linux/pw-cli.md

```diff
@@ -0,0 +1,16 @@
+# pw-cli
+
+> The PipeWire Command Line Interface.
+> More information: <https://docs.pipewire.org/page_man_pw_cli_1.html>.
+
+- List all nodes (sinks and sources) along with their IDs:
+
+`pw-cli list-objects Node`
+
+- Print information about an object with the specified ID:
+
+`pw-cli info {{4}}`
+
+- Print all objects' information:
```

Comment:
```suggestion
- Print all object information:
```

Reply:
It doesn't print all information about single object but information about every object. All nodes and devices.